### ISSUE TITLE
[FL autosuggest prototype] Icons misaligned when browser font size is changed

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#22513]
## Refinement questions
- Focus styles also differ (border radius, possibly more). Does it make sense to update those styles here as well? Or should a new ticket be created? 

## Describe the defect
Found as part of [ADE accessibility audit](https://github.com/department-of-veterans-affairs/va.gov-team/issues/120537#issuecomment-3347814120) of the location search box, Facility type select, custom select, and autosuggest fields on the [autosuggest prototype](https://staging.va.gov/find-locations/).

> **Issue: “X” buttons misaligned**
> **Field:** Location search box, autosuggest
> **Expected severity level:** 4
> **Expected experience standard:** User encounters components and patterns that meet best practices
> **Description:** “X” buttons within your custom <input> components are misaligned when the browser font size is increased:
> **How to recreate:** Using Chrome, go to the Settings, search for “font,” and increase the font size to “Very Large.”
> **Recommendation:** Is it possible to vertically align the “X,” rather than use absolute positioning?

Currently our 'X' icons and carets are misaligned when browser text size settings are enlarged. We want to match how [va-combo-box](https://design.va.gov/components/form/combo-box) displays when browser text settings are enlarged, so that the icons stay aligned.

## Notes
- [VADS combo box Storybook component](https://design.va.gov/storybook/?path=/docs/uswds-va-combo-box--docs). Good pattern to follow.
- [Location input and services autosuggest are custom comboxes](https://dsva.slack.com/archives/C0FQSS30V/p1759523788832689?thread_ts=1759522358.548369&cid=C0FQSS30V) and are therefore not inheriting expected styling from VADS inputs.
- This is _not_ an issue when you use standard browser zoom. Icons looked centered in Firefox, Chrome, and Safari.

## To Reproduce
1. Using Chrome, go to the Settings, search for “font,” and increase the font size to “Very Large.” 
2. Go to [autosuggest prototype](https://staging.va.gov/find-locations/)
3. Type into the <input> fields (location and services type).
4. See that the 'X' icons are misaligned. 
5. Also see that in the services type field the caret is misaligned.

## Screenshots
Current Facility locator inputs after changing Chrome font size setting to 'very large':
<img width="324" height="187" alt="Image" src="https://github.com/user-attachments/assets/0cae6d6f-fbc8-4dfc-97c2-e28d77899d20" />

VADS combo box after changing Chrome font size setting to 'very large':
<img width="497" height="126" alt="Image" src="https://github.com/user-attachments/assets/ce6b2844-8571-4a08-8978-2eff05f9c826" />

## AC / Expected behavior
- [ ] Icons are centered in the inputs even when browser text settings are enlarged for both the location and service type fields.

## Definition of done
- [ ] Code completed; PR created
- [ ] Review instance created/available; internal team notified to start reviews
- [ ] Code review (meets established quality and security standards)
- [ ] Design review (meets design & UX requirements)
- [x] A11y review (meets accessibility standards)
- [ ] Functional/feature review (meets acceptance criteria)
- [ ] Automated tests passed; PR approved
- [ ] Testing completed; dev team notified to deploy
- [ ] PO/PM/external verify in prod
- [ ] Documentation updated

**Comment by qjac** [comment]
Sorry @r-bartlett-gsa and @carlyMhayter. I didn't mean to close it! But it does look good to me. I reopened and put back in 'review' column.
